app/src/main/java/com/mailerdaemon/app/clubs/ClubDetailBottomSheet.kt

```diff
@@ -0,0 +1,152 @@
+package com.mailerdaemon.app.clubs
+
+import android.annotation.SuppressLint
+import android.os.Build
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.MotionEvent
+import android.view.MotionEvent.ACTION_BUTTON_PRESS
+import android.view.View
+import android.view.ViewGroup
+import androidx.swiperefreshlayout.widget.CircularProgressDrawable
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import com.mailerdaemon.app.BuildConfig
+import com.mailerdaemon.app.R
+import com.mailerdaemon.app.utils.ChromeTab
+import kotlinx.android.synthetic.main.fragment_club_detail.view.club_icon
+import kotlinx.android.synthetic.main.fragment_club_detail.view.club_des
+import kotlinx.android.synthetic.main.fragment_club_detail.view.club_fb
+import kotlinx.android.synthetic.main.fragment_club_detail.view.club_insta
+import kotlinx.android.synthetic.main.fragment_club_detail.view.club_members
+import kotlinx.android.synthetic.main.fragment_club_detail.view.club_name
+import kotlinx.android.synthetic.main.fragment_club_detail.view.club_web
+import kotlinx.android.synthetic.main.fragment_club_detail.view.club_youtube
+import kotlinx.android.synthetic.main.fragment_club_detail.view.web
+import org.json.JSONException
+import org.json.JSONObject
+import java.io.IOException
+import java.nio.charset.StandardCharsets
+
+@Suppress("DEPRECATION")
```

Comment:
What is deprecated? Can it be replaced with the newer method?

Reply:
setImageResource is deprecated. cant be replace even if I replace it shows the new one to be deprecated too